time stamp
int16[4] raw